test/custom.test.js

```diff
@@ -36,7 +36,8 @@ export default class App extends React.Component<Props, State> {
   }
 
   onChange = (selectedItem: Item) => {
```

Comment:
Same here, can't see that it's being used somewhere?

Reply:
Used for TypeScript, so I think we can leave it. I kinda leave this stuff up to the TS maintainers.